tests/std/tests/P1951R1_default_arguments_pair_forward_ctor/test.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <memory>
+#include <utility>
+
+using namespace std;
+
+int main() {

```

Comment:
This can be a compile only test. See a  `.compile.pass` file for an example. 

Should just need to rename the file and add a commen.